src/utils/get-options/quarantine.ts

```diff
@@ -0,0 +1,39 @@
+import baseGetOptions from './base';
+import QUARANTINE_OPTION_NAMES from '../../configuration/quarantine-option-names';
+import { RUNTIME_ERRORS } from '../../errors/types';
+import { GeneralError } from '../../errors/runtime';
+import { Dictionary } from '../../configuration/interfaces';
+
+
+function isQuarantineOption (option: string): option is QUARANTINE_OPTION_NAMES {
+    return Object.values(QUARANTINE_OPTION_NAMES).includes(option as QUARANTINE_OPTION_NAMES);
+}
+
+function validateQuarantineOptions (options: Dictionary<string | number> ): void {
+    const { retryCount, passCount } = options;
+
+    if (!passCount && retryCount < 3)
```

Comment:
Here we check the `passCount` variable. And if it does not exists or equal to 0 we throw the `invalidRetryCount` error. It looks there is a mistake here. I mean, that we should not throw throw `retryCount` error if we have incorrect `passCount`.

Reply:
The reason I implemented it this way is in the scenario where _retryCount_ is specified without _passCount_, then we want to ensure _retryCount_ is above the default quarantine threshold value (3).

```sh
testCafe chrome -q                // PassThreshold is 3, FailThreshold is 3
testcafe chrome -q retryCount=10  // PassThreshold is 3, FailThreshold is 8
testcafe chrome -q passCount=0    // Should fail, will add a test and validate this
testcafe chrome -q passCount=4    // Should fail, passCount > retryCount (default 3)
testcafe chrome -q retryCount=2   // Should fail, passCount (default 3) > retryCount
```

I can change the error to be something like: `passCount should be less than retryCount (retryCount value)`?

Reply:
Ok, I understand your idea. Thank you for explanation. What if we rewrite the code as follows:
```JS
function validateQuarantineOptions (options: Dictionary<string | number> ): void {
    const retryCount = options.retryCount || DEFAULT_TEST_RUN_THRESHOLD;
    const passCount  = options.passCount || DEFAULT_QUARANTINE_THRESHOLD;

    if (passCount > retryCount)
        throw new GeneralError(RUNTIME_ERRORS.invalidRetryCountValue, passCount);
}
```
Here I added default values for passCount and retryCount, so we do not need to check the `passCount` variable for undefined.

Could you please check this approach?
I would also prefer to import `DEFAULT_TEST_RUN_THRESHOLD` and `DEFAULT_QUARANTINE_THRESHOLD` constants. I think we can extract the Quarantine class from the `test-run-controller` file to the `quarantine` file and import these constants from the `quarantine` file.